### DISCUSSION TITLE
Jan 22

### DISCUSSION THREAD

**Discussion by ngsy** [GymHubCommunity/GymHub-FE#30] [DailyMeeting]
### ⚽️ 회의 내용

- 역할분배 
    - 남은 셋팅 분배 
    - 기간 설정 

### 🏀 오늘 할 일

  - 수: modal , wiki 
  - 수영: ga ,hotjar , mixpanel
  - 슬희: react-hook-form , 뒤로가기 기능 
  - 윤혁: style , component 폴더 구조 아토믹 디자인패턴 적용

### 🎱 TMI

- 윤혁: 오늘 엄청 춥다 나갈꺼면 옷 잘입으세용🧚‍♂️
- 수: PT 가기 싫다. 환불 받고싶다. 숙제를 못했다. PT 취소에 대한 이유 찾는 중
- 슬희: 심장? 병원을 가보실 예정, 병원 꼭 가보시길 ,오늘 기획안 오전까지 제출 해야 함 
- 수영: 카페를 갈까말까 고민 중입니다. 나갑니다 지방공간 사용비 알뜰살뜰
- 태훈: 8시에 일어나려하셨지만 10시에 일어나셨다 늦잠을 자버렸다... 9시에 일어나서 코테를 푸는데 극한상황..연출

**Comment by hanseulhee** [comment]
pt 숙제 안해서 어캐 됐대용?

**Reply by miniposi** [reply]
살아남았습니다..
